task_01/js.js

```diff
@@ -0,0 +1,74 @@
+document.getElementById('container').innerHTML = "<h2>Krestiki Noliki</h2>";
+const table = document.getElementById('gameField');
+table.onclick = function(event) {
+  const target = event.target;
+
+  // цикл двигается вверх от target к родителям до table
+  while (target !== table) {
+    if (target.tagName === 'TD') {
+      // нашли элемент, который нас интересует!
+      doMove(target);
+      return;
+    }
+    target = target.parentNode;
+  }
+
+  // возможна ситуация, когда клик был вне <td>
+  // если цикл дошёл до table и ничего не нашёл,
+  // то обработчик просто заканчивает работу
+}
+let stateMatrix = table.dataset.stateMatrix;
+stateMatrix = [[" "," "," "],[" "," "," "],[" "," "," "]];

```

Comment:
"what the hidden meaning of this? You create stateMatrix var with some value and in the next line, you change this line. Seems strange and undeeded"

Reply:
it was try to user data attribute w/o change the rest of code)))))) it's definitely failed, i am not sure how to use correct way data attributes for dynamically changing arrays, arrays or arrays, etc.